design-documents/graph-ql/coverage/customer/gift-registry.graphqls

```diff
@@ -128,40 +130,61 @@ input GiftRegistryDynamicAttributeInput {
     value: String!
 }
 
-type CreateGiftRegistryOutput {
-    gift_registry: GiftRegistry
+interface GiftRegistryErrorsInterface {
+    errors: [GiftRegistryItemError]!
+}
+
+interface GiftRegistryItemErrorInterface {
+    error: GiftRegistryItemError
```

Comment:
@pdohogne-magento I addressed your concerns, but this one
I do like grouping fields contained in GiftRegistryItemError under a type on a different node 'error'
in that case GiftRegistryItemError implements GiftRegistryItemErrorInterface could happen but doesn't have a lot of sense unless we use that on the entire schema in a more general way:

`GiftRegistryItemError implements ItemErrorInterface`

as for the GiftRegistryItemErrorInterface I suggest we find another name for it?